libraries/botbuilder-dialogs/tests/skillDialog.test.js

```diff
@@ -18,125 +18,143 @@ const { spy, stub } = require('sinon');
 const { Dialog, DialogTurnStatus, SkillDialog, DialogSet } = require('../');
 
 const DEFAULT_OAUTHSCOPE = 'https://api.botframework.com';
-const DEFAULT_GOV_OAUTHSCOPE = 'https://api.botframework.us';
-
-const defaultSkillDialogOptions = {
-    botId: 'botId',
-    conversationIdFactory: {},
-    conversationState: {},
-    skill: {},
-    skillHostEndpoint: {}
-};
-
-function typeErrorValidator(e, expectedMessage) {
-    ok(e instanceof TypeError);
-    strictEqual(e.message, expectedMessage);
-}
 
-describe('SkillDialog', function() {
+describe('SkillDialog', function () {
     this.timeout(3000);
 
-    describe('beginDialog should call skill', () => {
+    describe('beginDialog should call skill', function () {
         let BOTBUILDER = null;
         let BOTBUILDER_TESTING = null;
         // Use botbuilder for tests
         try {
             BOTBUILDER = require('../../botbuilder');
             BOTBUILDER_TESTING = require('../../botbuilder-testing');
-        } catch (err) {
-            console.warn('=====\nUnable to load botbuilder or botbuilder-testing module. "beginDialog should call skill" tests will not be run.\n');
+        } catch (_err) {
+            console.warn(
+                '=====\nUnable to load botbuilder or botbuilder-testing module. "beginDialog should call skill" tests will not be run.\n'
+            );
         }
 
-        if (BOTBUILDER !== null && BOTBUILDER_TESTING !== null) {
-            const { DialogTestClient } = BOTBUILDER_TESTING;
-
-            /**
-             * @remarks
-             * Port of BeginDialogShouldCallSkill from C#
-             * https://github.com/microsoft/botbuilder-dotnet/blob/41120728b22c709ec2d9247393505fdc778b2de1/tests/Microsoft.Bot.Builder.Dialogs.Tests/SkillDialogTests.cs#L47-L49
-             * @param {string} deliveryMode 
-             */
-            async function beginDialogShouldCallSkill(deliveryMode, useCreateSkillConversationId = false) {
```

Comment:
Note: This is difficult to read in the diff, but I moved this function definition outside of the `if` statement.

Reply:
Thanks for adding the note!